test/gurka/test_shortcut.cc

```diff
@@ -74,6 +74,57 @@ TEST(Shortcuts, CreateInvalid) {
   EXPECT_ANY_THROW(gurka::findEdgeByNodes(graph_reader, layout, "C", "A"));
 }
 
+// Here no shortcuts are created.
+// The grid size should be greater than the max length that allows shortcuts to be created
+TEST(Shortcuts, CreateTooLong) {
+  constexpr double gridsize = 1500000;
+  // Maximum edge length is    16777215
```

Comment:
can you indent this number better to align with the number over it? so it's easier to see that it's a full magnitude difference